tests/providers/aws/lib/mutelist/aws_mutelist_test.py

```diff
@@ -1132,7 +1132,7 @@ def test_is_muted_tags(self):
         }
         mutelist = AWSMutelist(mutelist_content=mutelist_content)
 
-        assert mutelist.is_muted(
```

Comment:
Why this test changes?

Reply:
This test was checking for a bad behaviour, in this case we have two tags on the mutelist so passing only one on the resource_tags fiel should not be muted.